server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -22,6 +22,7 @@ grant codeBase "${codebase.elasticsearch-secure-sm}" {
 grant codeBase "${codebase.elasticsearch}" {
   // needed for loading plugins which may expect the context class loader to be set
   permission java.lang.RuntimePermission "setContextClassLoader";
+  permission java.lang.RuntimePermission "accessClassInPackage.jdk.internal.reflect";
```

Comment:
Can you please add a comment here explaining what needs this permission?

Reply:
this should be removed. We added this in a separate commit earlier (with a comment)